contracts/fatex/FateReward/FateRewardControllerV3.sol

```diff
@@ -90,6 +90,7 @@ contract FateRewardControllerV3 is IFateRewardController, LPWithdrawFee {
         address _vault,
         IFateRewardController[] memory _oldControllers,
         IMockLpTokenFactory _mockLpTokenFactory,
+        uint256 _epochStartBlock,
         uint256 _epochEndBlock
     ) public {
```

Comment:
We discussed the start/end blocks being moved to the reward schedule contract to make the reward controller v3 more reusable across epochs